neuror/main.py

```diff
@@ -284,9 +287,12 @@ def __init__(self,
                                 for section in self.neuron.iter())
 
         self.info = dict()
-        apical_section_id, _ = apical_point_section_segment(self.neuron)
+        if apical_point:
+            apical_section_id, _ = point_to_section_segment(self.neuron, apical_point)
+        else:
+            apical_section_id, _ = apical_point_section_segment(self.neuron)
         if apical_section_id:
-            self.apical_section = self.neuron.sections[apical_section_id]
+            self.apical_section = self.neuron.sections[apical_section_id - 1]
```

Comment:
why -1 here?

Reply:
I think it's a bug  because of  a +1 in  morph-tool (for some morphio reason I don't really know), but we need to check with @wizmer 
I saw it because on my  test morph,  the apical point was on last section, so apical_section_id=194, which crashed this line, as len(self.neuron.section) = 194  ;)

Reply:
So in `morph_tool.spatial.point_to_section_segment` there is +1, then in `morph_tool.apical_point.apical_point_section_segment` there is -1, then here there is -1 :exploding_head: :laughing:

Reply:
arg so this -1 +1 is taken care of in morph-tool lol. I just need to move the -1 in my  if haha good catch!